doc/examples/transform/plot_transform_types.py

```diff
@@ -0,0 +1,142 @@
+"""
+===================================
+Types of homographies
+===================================
+
+`Homographies <https://en.wikipedia.org/wiki/Homography>`_
+are transformations of a Euclidean space that preserve the alignment of points.
+Specific cases of homographies correspond to the conservation of more properties,
+such as parallelism (affine transformation), shape (similar transformation) or distances (Euclidean transformation).
```

Comment:
@emmanuelle can you align these paragraphs to 80c? The content seems fine to me now. =)